pages/task/list/views/table.jsx

```diff
@@ -54,7 +55,7 @@ const formatters = {
           }
           {
             continuation && (
-              firstExpiry
+              firstExpiry && isValid(firstExpiry)
```

Comment:
It's probably worth rolling this into the `firstExpiry` bit above in case there's one with a valid date and another without.